### PR TITLE
Mask API key for Anyscale LLM

### PR DESCRIPTION
**Pull request by aidoskanapyanov**
Description: Add masking of API Key for Anyscale LLM when printed.
Issue: #12165 
Dependencies: None
Tag maintainer: @eyurtsev

### CODE REVIEW

**Comment by eyurtsev** [comment]
Hi @aidoskanapyanov looks good,  you need to invoke `.get_secret_value() ` in the appropriate places to actually use the unmasked value when the secret needs to be sent with its raw value.

See: https://github.com/langchain-ai/langchain/pull/12377/files#diff-2f8314a1f61a50eb15fc34e9c7a0be1ad6f8be474ddd66321f4e65e7d1746143R186

**Comment by aidoskanapyanov** [comment]
Got it, I'll fix it in a moment 👍

File: libs/langchain/langchain/llms/anyscale.py [hunk 1]
```diff
@@ -94,24 +95,33 @@ def validate_environment(cls, values: Dict) -> Dict:
...
-            values, "anyscale_api_key", "ANYSCALE_API_KEY"
+        values["anyscale_api_key"] = convert_to_secret_str(
+            get_from_dict_or_env(values, "anyscale_api_key", "ANYSCALE_API_KEY")
         )
         try:
             import openai
 
-            ## Always create ChatComplete client, replacing the legacy Complete client
-            values["client"] = openai.ChatCompletion
+            openai.api_key = values["anyscale_api_key"].get_secret_value()
```
Comment (eyurtsev) [inline libs/langchain/langchain/llms/anyscale.py:104]: We shouldn't do this -- it updates a global value for openai.api_key which can lead to issue is openai is used across models with different keys

**Comment by eyurtsev** [comment]
Ready to merge after tests pass

**Pull request merged**